dev/ScrollViewer/ScrollViewer.cpp

```diff
@@ -483,7 +487,13 @@ void ScrollViewer::OnGotFocus(winrt::RoutedEventArgs const& args)
         m_focusInputDeviceKind == winrt::FocusInputDeviceKind::Mouse ||
         m_focusInputDeviceKind == winrt::FocusInputDeviceKind::Pen;
 
-    ShowIndicators();
+    UpdateVisualStates(
+        true  /*useTransitions*/,
```

Comment:
This taking a sea of bool is... not my favorite. Might as well add a struct options type at this point so that it's mandatory. Example:

UpdateVisualStates options {};
options.useTransitions = true;
options.showIndiators = true;
options.updateScrollControllersAutoHiding = true;
options.onlyForAutoHidingScrollControllers = true;
UpdateVisualStates(options);

Reply:
I don't think this is worthy of a new struct.   It keeps UpdateVisualStates(false /*useTransitions*/) simple.